packages/common/api.js

```diff
@@ -198,6 +198,8 @@ export const deleteTeam = teamId => callApi.delete(`/teams/${teamId}`);
 
 export const getTeam = teamId => callApi.get(`/teams/${teamId}`);
 
+export const pinTeam = teamId => callApi.post(`/teams/${teamId}/pinned`);
```

Comment:
How I can unpin the team? I don't see such api. 
@aburov-4xxi alos, why `/teams/${teamId}/pinned`? Should be the action of a team or an entity, for ex. `/teams/${teamId}/pin` but `pinned` is the property of the team.

Reply:
The same endpoint is used to unpin a team

Reply:
Updated api calss